src/transformers/models/luke/modeling_luke.py

```diff
@@ -0,0 +1,1331 @@
+            Indices of positions of each input entity in the position embeddings. Selected in the range ``[0,
+            config.max_position_embeddings - 1]``.
+
+        inputs_embeds (:obj:`torch.FloatTensor` of shape :obj:`({0}, hidden_size)`, `optional`):
+            Optionally, instead of passing :obj:`input_ids` you can choose to directly pass an embedded representation.
+            This is useful if you want more control over how to convert :obj:`input_ids` indices into associated
+            vectors than the model's internal embedding lookup matrix.
+
+        head_mask (:obj:`torch.FloatTensor` of shape :obj:`(num_heads,)` or :obj:`(num_layers, num_heads)`, `optional`):
+            Mask to nullify selected heads of the self-attention modules. Mask values selected in ``[0, 1]``:
+
+            - 1 indicates the head is **not masked**,
+            - 0 indicates the head is **masked**.
+
+        output_attentions (:obj:`bool`, `optional`):
+            Whether or not to return the attentions tensors of all attention layers. See ``attentions`` under returned
+            tensors for more detail.
+        output_hidden_states (:obj:`bool`, `optional`):
+            Whether or not to return the hidden states of all layers. See ``hidden_states`` under returned tensors for
+            more detail.
+        return_dict (:obj:`bool`, `optional`):
+            Whether or not to return a :class:`~transformers.file_utils.ModelOutput` instead of a plain tuple.
+"""
+
+
+@add_start_docstrings(
+    "The bare LUKE Model transformer outputting raw hidden-states for both word tokens and entities without any specific head on top.",
+    LUKE_START_DOCSTRING,
+)
+class LukeModel(LukePreTrainedModel):
+
+    _keys_to_ignore_on_load_missing = [r"position_ids"]
+
+    def __init__(self, config, add_pooling_layer=True):
+        super().__init__(config)
+        self.config = config
+
+        self.embeddings = LukeEmbeddings(config)
+        self.embeddings.token_type_embeddings.requires_grad = False
+        self.entity_embeddings = LukeEntityEmbeddings(config)
+        self.encoder = LukeEncoder(config)
+
+        self.pooler = LukePooler(config) if add_pooling_layer else None
+
+        self.init_weights()
+
+    def get_input_embeddings(self):
+        return self.embeddings.word_embeddings
+
+    def set_input_embeddings(self, value):
+        self.embeddings.word_embeddings = value
+
+    def get_entity_embeddings(self):
+        return self.entity_embeddings.entity_embeddings
+
+    def set_entity_embeddings(self, value):
+        self.entity_embeddings.entity_embeddings = value
+
+    def _prune_heads(self, heads_to_prune):
+        raise NotImplementedError("LUKE does not support the pruning of attention heads")
+
+    @add_start_docstrings_to_model_forward(LUKE_INPUTS_DOCSTRING.format("(batch_size, sequence_length)"))
```

Comment:
```suggestion
    @add_start_docstrings_to_model_forward(LUKE_INPUTS_DOCSTRING.format("batch_size, sequence_length"))
```
The ( ) should be in the docstring.